test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/001_GetVehicleData_fuelRange_all_params.lua

```diff
@@ -0,0 +1,27 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Please notice target branch `feature/fuel_info_related_vehicle_date` is incorrect. Typo `_date`.
We have an agreement to have the same branch name across all repositories (`sdl_core`, `sdl_hmi` etc.)

Reply:
@dboltovskyi target branch was changed